ATP.BusinessLogicLayer.Test/ServicesTest/ManagerServiceTest.cs

```diff
@@ -3,16 +3,9 @@
 
 namespace ATP.BusinessLogicLayer.Test.ServicesTest;
 public class ManagerServiceTest
-{  
-    private static readonly List<FlightDomainModel> Flights = new List<FlightDomainModel>
-    {
-        new FlightDomainModel(1, 100.0, "USA", "UK", new DateTime(2024, 03, 10), "JFK", "LHR", FlightClass.Economy),
-        new FlightDomainModel(2, 500.0, "UK", "France", new DateTime(2024, 03, 15), "LHR", "CDG", FlightClass.Business),
-        new FlightDomainModel(3, 300.0, "Germany", "Italy", new DateTime(2024, 03, 20), "TXL", "FCO", FlightClass.First),
-        new FlightDomainModel(4, 200.0, "Spain", "Greece", new DateTime(2024, 03, 25), "MAD", "ATH", FlightClass.Economy),
-        new FlightDomainModel(5, 400.0, "Australia", "Japan", new DateTime(2024, 03, 30), "SYD", "HND", FlightClass.Business),
-        new FlightDomainModel(6, 600.0, "China", "Singapore", new DateTime(2024, 04, 05), "PEK", "SIN", FlightClass.First)
-    };
+{
+    static string filePath = "C:\\Users\\hp\\Desktop\\C#\\ATP.BusinessLogicLayer.Test\\ServicesTest\\TestFlights.json";
```

Comment:
Lets try something more relative, using appsetting.json and something like this path
```suggestion
    static string filePath = "..\\ATP.BusinessLogicLayer.Test\\ServicesTest\\TestFlights.json";
```